src/components/common/request-details.vue

```diff
@@ -3,59 +3,77 @@
     <section>
       <h4>Requested Name <span v-if="nameChoices.name2">Choices</span></h4>
       <ul class="pl-0">
-        <li v-if="!nameChoices.name2">{{nameChoices.name1}} {{nameChoices.designation1}}</li>
+        <li v-if="!nameChoices.name2">
+          {{ nameChoices.name1 }} {{ nameChoices.designation1 }}
+        </li>
         <template v-else>
-          <li v-if="nameChoices.name1">1. {{nameChoices.name1}} {{nameChoices.designation1}}</li>
-          <li v-if="nameChoices.name2">2. {{nameChoices.name2}} {{nameChoices.designation2}}</li>
-          <li v-if="nameChoices.name3">3. {{nameChoices.name3}} {{nameChoices.designation3}}</li>
+          <li v-if="nameChoices.name1">
+            1. {{ nameChoices.name1 }} {{ nameChoices.designation1 }}
+          </li>
+          <li v-if="nameChoices.name2">
+            2. {{ nameChoices.name2 }} {{ nameChoices.designation2 }}
+          </li>
+          <li v-if="nameChoices.name3">
+            3. {{ nameChoices.name3 }} {{ nameChoices.designation3 }}
+          </li>
         </template>
       </ul>
     </section>
 
     <v-row>
-      <v-col col="6">
+      <v-col class="col">
```

Comment:
Hmm, weird auto-fix.

I will test this before merging.

Reply:
New on the left, old on the right. Works fine.

![image](https://github.com/bcgov/namerequest/assets/10002889/5f6a77e4-c451-435f-8806-25ba7661e6e6)